### DISCUSSION TITLE
When I tried to add another Git account, it wasn't added and I can't see its repositories.

### DISCUSSION THREAD

**Discussion by webtechlavv** [vercel/community#5327] [Help]
### Summary

I was adding another Git account to Vercel, but the account wasn't added and I can't see any of its repositories. Is there anything I can do to troubleshoot this?
![image](https://github.com/vercel/community/assets/152217961/c34ad871-6f69-4ed3-8b9d-38aa8cfbb147)

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/152217961/60873534-98fd-4bc7-8626-11414d4d9af2)

**Comment by swarnava** [comment]
Hi, 

You can add 1 GitHub account per Vercel account for now. Adding multiple account isn't possible at this moment.

**Reply by azeem-io** [reply]
Then why is there an option to add another Github account?
